### PR TITLE
Added DeclareEqualNormalizeFixer

### LINKED ISSUE #2064: Add DeclareEqualNormalizeFixer
```diff
-<?php declare(ticks =  1);
+<?php declare(ticks=1);
```

better name could be nice as well ;)

### PR DESCRIPTION
**Pull request by keradus**
Closes #2064

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/DeclareEqualNormalizeFixer.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class DeclareEqualNormalizeFixer extends AbstractFixer
```
Comment (SpacePossum) [inline Symfony/CS/Fixer/Symfony/DeclareEqualNormalizeFixer.php]: maybe declare final for master requirement?

File: Symfony/CS/Tests/Fixer/Symfony/DeclareEqualNormalizeFixerTest.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class DeclareEqualNormalizeFixerTest extends AbstractFixerTestBase
```
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/DeclareEqualNormalizeFixerTest.php]: final and @internal?

**Comment by SpacePossum** [comment]
:+1:  LGTM, thanks! just two minor comments :)

**Comment by keradus** [comment]
done ;)

**Comment by SpacePossum** [comment]
thanks & :ship:  :)

**Comment by GrahamCampbell** [comment]
👍

**Comment by SpacePossum** [comment]
could you take out the method;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/Strict/DeclareStrictTypesFixer.php#L111

otherwise I'll PR it after this one, we no longer need the method as your fixer does the same :)

**Comment by keradus** [comment]
Good point.
But please do it as separate PR - this one is for 1.12, the one you mention is for 2.x ;)

**Comment by SpacePossum** [comment]
ah yes, course :)
I'll put in on my 2.x todo list :+1:

**Comment by keradus** [comment]
merged to 1.12 and master.
@SpacePossum please open the follow up PR for master branch only

**Pull request closed without merging**